tests/functional/test_install_index.py

```diff
@@ -65,9 +65,7 @@ def test_file_index_url_quoting(script, data):
     result = script.pip(
         'install', '-vvv', '--index-url', index_url, 'simple'
     )
-    assert (script.site_packages / 'simple') in result.files_created, (
-        str(result.stdout)
-    )
-    assert (
+    result.did_create(script.site_packages / 'simple')
```

Comment:
```suggestion
    result.did_create(script.site_packages / 'simple', str(result.stdout))
```

Reply:
No -- this information is included in `str(result)`.